.github/bors.toml

```diff
@@ -0,0 +1,2 @@
+delete_merged_branches = true
+status = ["ci"]
```

Comment:
What is this file about? I don't think I saw it before. There's also still `bors.toml` in the root.

Reply:
this is the same file as the `bors.toml` file that was in the root. the bors.toml can be either in the root or in the .github folder (apparently?). I can put back in the root to reduce churn.